internal/driver/checkstatuses.go

```diff
@@ -37,10 +35,13 @@ func (d *Driver) checkStatuses() {
 	}
 }
 
-// testConnectionAuth will try to send a command to a camera using authentication
-// and return a bool indicating success or failure
+// testConnectionMeethods will try to determine the state using different device calls
+// and return the most accurate status
+// Higher degrees of connection are tested first, becuase if they
+// succeed, the lower levels of connection will too
 func (d *Driver) testConnectionMethods(device sdkModel.Device) (status string) {
-	// sends get capabilities command to device (does not require credentials)
+
+	// creates client for device (does not require credentials)
```

Comment:
you should put back in that it calls a get capabilities as part of that. just creating the client doesnt make me assume it can actually connect.